--- a/neutron/cmd/destroy_patch_ports.py
+++ b/neutron/cmd/destroy_patch_ports.py
@@ -39,6 +39,7 @@

 class PatchPortCleaner(object):
     def __init__(self, config):
+        LOG.debug("Get OVS bridge mappings")
         mappings = helpers.parse_mappings(config.OVS.bridge_mappings)
         self.bridges = [ovs_lib.OVSBridge(bridge)
                         for bridge in mappings.values()]
@@ -52,6 +53,7 @@
             return
         for bridge in self.bridges:
             try:
+                LOG.debug("Remove patch port from bridge %s", bridge.br_name)
                 self._remove_patch_ports_from_int_br(bridge)
             except Exception as e:
                 LOG.error("Failed to remove patch port from bridge %s: %s",
@@ -69,12 +71,15 @@
     def flows_configured(self):
         """Return True if the integration bridge has flows already configured.
         """
+        LOG.debug("Get configured flows for integration bridge %s",
+                  self.int_br.br_name)
         return bool(self.int_br.dump_flows_for(table=constants.CANARY_TABLE))


 def main():
     common_config.init(sys.argv[1:])
     ovs_conf.register_ovs_agent_opts()
+    common_config.setup_logging()
     port_cleaner = PatchPortCleaner(cfg.CONF)
     port_cleaner.destroy_patch_ports()

